site/public/css/course-materials.css

```diff
@@ -32,3 +32,7 @@ a.popout-item > .fas:hover {
     display: flex;
     flex-wrap: nowrap;
 }
+
+.btn.btn-primary {
+    margin-top: 4px;
+}
```

Comment:
This seems like the wrong fix for this problem.  What if there was a different primary button somewhere else on the page?  It would also get this new margin.  It would be better to add a new class to each of the buttons, or better yet, use flexbox properties to do the wrapping cleanly.